src/app/fdctl/run/tiles/fd_replay.c

```diff
@@ -55,12 +55,14 @@
 #define MAX_TXNS_PER_REPLAY ( ( FD_SHRED_MAX_PER_SLOT * FD_SHRED_MAX_SZ) / FD_TXN_MIN_SERIALIZED_SZ )
 
 
-#define STORE_IN_IDX   (0UL)
-#define PACK_IN_IDX    (1UL)
+#define STORE_IN_IDX    (0UL)
+#define PACK_IN_IDX     (1UL)
+#define RESTART_IN_IDX  (2UL)
 
-#define NOTIF_OUT_IDX  (0UL)
-#define SENDER_OUT_IDX (1UL)
-#define POH_OUT_IDX    (2UL)
+#define NOTIF_OUT_IDX   (0UL)
+#define SENDER_OUT_IDX  (1UL)
+#define POH_OUT_IDX     (2UL)
```

Comment:
It seems that many POH out links are added, not just one, so that `RESTART_OUT_IDX` may not be 3UL